src/test/mempool_tests.cpp

```diff
@@ -466,10 +466,15 @@ BOOST_AUTO_TEST_CASE(MempoolSizeLimitTest)
     BOOST_CHECK(pool.exists(tx2.GetHash()));
     BOOST_CHECK(pool.exists(tx3.GetHash()));
 
-    pool.TrimToSize(GetVirtualTransactionSize(CTransaction(tx1))); // mempool is limited to tx1's size in memory usage, so nothing fits
+    std::vector<COutPoint> vNoSpendsRemaining;
+    pool.TrimToSize(GetVirtualTransactionSize(CTransaction(tx1)), &vNoSpendsRemaining); // mempool is limited to tx1's size in memory usage, so nothing fits
     BOOST_CHECK(!pool.exists(tx1.GetHash()));
     BOOST_CHECK(!pool.exists(tx2.GetHash()));
     BOOST_CHECK(!pool.exists(tx3.GetHash()));
+    // This vector should only contain 'root' (not unconfirmed) outpoints
```

Comment:
I don't think we use "root" anywhere else to refer to confirmed outpoints
```suggestion
    // This vector should only contain confirmed outpoints
```